torch/nn/modules/instancenorm.py

```diff
@@ -11,6 +11,22 @@ def __init__(self, num_features, eps=1e-5, momentum=0.1, affine=False,
     def _check_input_dim(self, input):
         return NotImplemented
 
+    def _load_state_dict_key_mismatch(self, full_name, name, is_missing):
+        if not is_missing and not self.track_running_stats and \
+                name in ('running_mean', 'running_var'):
+            raise KeyError(
+                'Unexpected running stats buffer in state_dict ("{name}") for '
+                '{klass} with track_running_stats=False. If you are trying to '
+                'load a checkpoint saved before 0.4.0, this may be expected '
+                'because {klass} does not track running stats by default '
+                'anymore since 0.4.0. If running stats are not used, set '
+                'strict=False in load_state_dict. Otherwise, set '
```

Comment:
I think it's better to recommend removing those entries from `load_state_dict`. Disabling strict mode can silently accept other issues